nav.php

```diff
@@ -2,31 +2,27 @@
 <?php
 if(session_id() == ''){session_start();}
 if(isset($_SESSION['login_user'])){?>
-    <nav>
-    <div class="mainlinks">
-        <li><a href="index.php">Home</a></li>
-        <li><a href="your_library.php">Your Library</a></li>
-        <li><a href="form.php">Contact</a></li>
-        <div class="dropdown">
-            <button class="dropbtn">Hi <?php echo $_SESSION['login_user']; ?></button>
-            <div class="dropdown-content">
-                <a href="settings.php">Settings</a>
-                <a href="logout.php">Logout</a>
+            <nav>
+            <div class="mainlinks">
+                <li><a href="index.php">Home</a></li>
+                <li><a href="your_library.php">Your Library</a></li>
+                <li><a href="form.php">Contact</a></li>
+                <div class="dropdown">
+                    <button class="dropbtn">Hi <?php echo $_SESSION['login_user']; ?></button>
+                    <div class="dropdown-content">
+                        <?php 
+                        if ($_SESSION["login_user"] == "Graham"){
+                            echo"<a href='settings.php'>Settings</a>";
+                        }
+                        echo "<a href='logout.php'>Logout</a>";?>
+                    </div>
```

Comment:
Nav bar displays a greeting for the user logged in and now check if the user signed in is Graham.